src/components/app-configure/ConfigureSidebar.vue

```diff
@@ -1,54 +1,273 @@
 <script lang="ts" setup>
+import { ref } from 'vue'
+import { useRouter } from 'vue-router'
+
+import ArcanaLogo from '@/assets/iconography/arcana-dark-vertical.svg'
+import arrowIcon from '@/assets/iconography/arrow.png'
+import profileIcon from '@/assets/iconography/profile.png'
 import VCard from '@/components/lib/VCard/VCard.vue'
 import VCardButton from '@/components/lib/VCardButton/VCardButton.vue'
 import VStack from '@/components/lib/VStack/VStack.vue'
+import { useAppsStore, type AppId } from '@/stores/apps.store'
+import { useAuthStore } from '@/stores/auth.store'
 import { CONFIGURE_TABS, type ConfigureTabType } from '@/utils/constants'
 
+const authStore = useAuthStore()
+const appsStore = useAppsStore()
+const showConfigureSubmenu = ref(false)
+const showAppsList = ref(false)
+const router = useRouter()
+const emit = defineEmits(['switch-tab'])
+
+function onClickOfMenu(tab: string) {
+  // emit('switch-tab', tab)
+  if (tab === 'configure') {
+    showConfigureSubmenu.value = !showConfigureSubmenu.value
+  }
+  console.log({ tab })
+}
+
+function onLogoClick() {
+  router.push('/')
+}
+
+function onAppClick(appId: AppId) {
+  appsStore.setSelectedAppId(appId)
+  router.push({ name: 'AppDetails', params: { appId } })
+}
+
 type ConfigureProps = {
   currentTab?: ConfigureTabType
 }
 
 withDefaults(defineProps<ConfigureProps>(), {
-  currentTab: 'general',
+  currentTab: 'dashboard',
 })
-
-const emit = defineEmits(['switch-tab'])
 </script>
 
 <template>
   <aside class="configure-sidebar">
     <VCard class="configure-sidebar-card">
-      <VStack direction="column" gap="2rem" class="configure-tabs flex-grow">
+      <button class="logo" @click.stop="onLogoClick">
+        <img :src="ArcanaLogo" alt="Arcana Logo" />
+      </button>
+      <VStack direction="column" gap="1rem" class="configure-tabs flex-grow">
+        <VStack class="apps-name__container">
+          <button
+            class="flex app-name__container"
+            @click="showAppsList = !showAppsList"
+          >
+            <label>{{ appsStore.selectedApp?.name }}</label>
+            <img
+              v-if="appsStore.apps.length > 1"
+              :src="arrowIcon"
+              alt="arrow-icon"
+              class="arrow-icon"
+              :class="{
+                'arrow-icon--active': showAppsList,
+              }"
+            />
+          </button>
+          <div
+            v-if="appsStore.apps.length > 1 && showAppsList"
+            class="apps-name__list-container"
+          >
+            <VCardButton
+              v-for="app in appsStore.apps"
+              :key="app.name"
+              class="apps-name__list-item"
+              :active="appsStore.selectedAppId === app.id"
```

Comment:
Can use `useAppId()` to get currently selected app Id instead of storing it in appStore